modules/sfp_pastebin.py

```diff
@@ -20,12 +20,32 @@ class sfp_pastebin(SpiderFootPlugin):
     """PasteBin:Footprint,Investigate,Passive:Leaks, Dumps and Breaches:apikey:PasteBin scraping (via Google) to identify related content."""
 
     meta = {
-		'name': "PasteBin",
-		'summary': "PasteBin scraping (via Google) to identify related content.",
-		'flags': [ "apikey" ],
-		'useCases': [ "Footprint", "Investigate", "Passive" ],
-		'categories': [ "Leaks, Dumps and Breaches" ]
-	}
+        'name': "PasteBin",
+        'summary': "PasteBin scraping (via Google) to identify related content.",
+        'flags': [ "apikey" ],
+        'useCases': [ "Footprint", "Investigate", "Passive" ],
+        'categories': [ "Leaks, Dumps and Breaches" ],
+        'dataSource': {
+            'website': "https://pastebin.com/",
+            'model': "FREE_AUTH_LIMITED",
+            'references': [
+                "https://pastebin.com/doc_api",
+                "https://pastebin.com/faq"
+            ],
+            'apiKeyInstructions': [
+                "Visit pastebin.com",
+                "Register a free account",
+                "Navigate to pastebin.com/doc_api",
+                "The API key is listed under 'Your Unique Developer API Key'"
```

Comment:
Actually it's the Google search API they need to obtain.

Reply:
So, instructions for getting google custom search api?

Reply:
Yep

Reply:
Updated